### PR TITLE
James amends

### PR DESCRIPTION
**Pull request by MCooke**

### CODE REVIEW

File: library/entry-meta.php [hunk 1]
```diff
@@ -9,8 +9,8 @@
 
 if ( ! function_exists( 'foundationpress_entry_meta' ) ) :
 	function foundationpress_entry_meta() {
-		echo '<time class="updated" datetime="'. get_the_time( 'c' ) .'">'. sprintf( __( 'Posted on %s at %s.', 'foundationpress' ), get_the_date(), get_the_time() ) .'</time>';
-		echo '<p class="byline author">'. __( 'Written by', 'foundationpress' ) .' <a href="'. get_author_posts_url( get_the_author_meta( 'ID' ) ) .'" rel="author" class="fn">'. get_the_author() .'</a></p>';
+		echo '<div class="">'. __( '', 'foundationpress' ) .' <a href="'. get_author_posts_url( get_the_author_meta( 'ID' ) ) .'" rel="author" class="fn">'. get_the_author() .'</a></div>';
```
Comment (lukefrake) [inline library/entry-meta.php:12]: What's this doing?

File: index.php [hunk 2]
```diff
@@ -17,7 +17,7 @@
 get_header(); ?>
 
 <div id="page" role="main">
-	<article class="main-content">
+	<article id="content" class="main-content">
```
Comment (lukefrake) [inline index.php:20]: What is this ID for? I can't see it being used?

File: library/entry-meta.php (hunk 1 above)
Comment (MCooke) [inline library/entry-meta.php:12]: Blog post meta, both in post and in list display

File: index.php (hunk 2 above)
Comment (MCooke) [inline index.php:20]: For Foundation Sticky, needs ID to use for stickyness boundries

**Comment by lukefrake** [comment]
:+1:

**Pull request merged**